hw/mcu/stm/stm32h7xx/src/clock_stm32h7xx.c

```diff
@@ -66,7 +66,7 @@ SystemClock_Config(void)
     __HAL_PWR_VOLTAGESCALING_CONFIG(MYNEWT_VAL(STM32_CLOCK_VOLTAGESCALING_CONFIG));
 
     osc_init.OscillatorType = RCC_OSCILLATORTYPE_NONE;
-
+    __HAL_RCC_PLL_PLLSOURCE_CONFIG(RCC_PLLSOURCE_HSE);
```

Comment:
This setting is already set with `osc_init.PLL.PLLSource = RCC_PLLSOURCE_HSE;`

Reply:
👍